modules/UI/videolayout/RemoteVideo.js

```diff
@@ -164,7 +163,9 @@ RemoteVideo.prototype._generatePopupContent = function () {
     let remoteControlState = null;
     let onRemoteControlToggle;
 
-    if (this._supportsRemoteControl) {
+    if (this._supportsRemoteControl
+        && ((!APP.remoteControl.active && !this._isRemoteControlSessionActive)
+            || APP.remoteControl.controller.activeParticipant === this.id)) {
```

Comment:
Is this._isRemoteControlSessionActive necessary? Could you get the same result by leaving the state in APP.remoteControl.controller.activeParticipant? (I suppose the same could be said about audio muted, which would not be for this changeset.)